uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
+            with Session() as session:
                 return session.guess_attendee_watchentry(self)
```

Comment:
I did the above in master, but you'll get the same error in your branch.

Well... technically you won't get an error because ``Attendee.banned`` will literally never return the result of ``session.guess_attendee_watchentry`` because ``listify(self.watch_list)`` will return ``[None]`` if there is no associated watchlist entry.  So using ``listify`` did technically prevent the error, but only by way of introducing an new bug :(